gnomad_qc/v5/resources/constants.py

```diff
@@ -1,6 +1,6 @@
 """Script containing version and release constants."""
 
 VERSIONS = ["5.0"]
-WORKSPACE_BUCKET = "gs://fc-secure-b25d1307-7763-48b8-8045-fcae9caadfa1"
+WORKSPACE_BUCKET = "fc-secure-b25d1307-7763-48b8-8045-fcae9caadfa1"
```

Comment:
does this line in resources/meta now need the gs:// added back in?
        path=f"{WORKSPACE_BUCKET}/v5.0/metadata/gnomad.v5.0.project_meta.ht"

Reply:
Yes, added it back!